cargo-near-build/src/near/docker_build/subprocess_step.rs

```diff
@@ -83,7 +89,12 @@ pub fn run(
             docker_args.extend(vec![&docker_image, "/bin/bash", "-c"]);
 
             docker_args.push(&shell_escaped_cargo_cmd);
-            tracing::debug!("docker command : {:#?}", docker_args);
+            tracing::info!(
```

Comment:
Second thiught, this doesn't need to be reverted to debug, as it's quite adequate verbosity for --teach-me, and RUST_LOG=info in container won't trigger this line, as it happens only outside of container. But i somehow got confused it would be triggered 😕

Reply:
It probably can be excluded from RUST_LOG=info and included into --teach_me by adjusting near_teach_me=debug but it's probably too much fuss around single debug statement

Reply:
I won't change it